tests/test-utlis.tsx

```diff
@@ -2,15 +2,26 @@ import { ReactElement, ReactNode } from "react";
 import { ThemeProvider } from "styled-components";
 import { theme } from "../src/theme/theme";
 import { render, RenderOptions } from "@testing-library/react";
+import { BrowserRouter } from "react-router-dom";
+import { QueryClientProvider } from "@tanstack/react-query";
+import { QueryClient } from "@tanstack/react-query";
 
 export const customRender = (
   ui: ReactElement,
   options?: Omit<RenderOptions, "wrapper">
 ) => {
-  const ThemeProviderWrapper = ({ children }: { children: ReactNode }) => {
-    return <ThemeProvider theme={theme}>{children}</ThemeProvider>;
+  const queryClient = new QueryClient();
```

Comment:
you already have a client instance defined somewhere - why not use it? This will ensure you're using the same configuration for the app and for testing it.

Reply:
I've client instance defined in the `App` but I don't know how I would use the same one for tests? Move `const queryClient = new QueryClient();` to a separate file and export it in order to use it in the App file and test-utils? I was also reading this article -https://tkdodo.eu/blog/testing-react-query and following a bit example given there. He was saying "I prefer to give each test its own QueryClientProvider and create a new QueryClient for each test. That way, tests are completely isolated from each other.". But not sure if that refers to the same thing you're mentioning here?